workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/load_diagnostic_data.py

```diff
@@ -87,15 +87,29 @@ def _load_prognostic_run_3d_output(url: str):
     prognostic_3d_output = [
         item
         for item in fs.ls(url)
-        if item.endswith("3d.zarr") or item.endswith("state_after_timestep.zarr")
+        if item.endswith("diags_3d.zarr") or item.endswith("state_after_timestep.zarr")
```

Comment:
`"diags_3d.zarr"` is the name that the default example saves ML tendencies and physics tendencies to.